vocabularies/gsq-sites.ttl

```diff
@@ -0,0 +1,139 @@
+    skos:broader gsqst:field-site ;
+    skos:definition "A field site from which a fossil sample is taken or an observation made."@en ;
+    skos:inScheme gsqst:conceptScheme ;
+    skos:prefLabel "Fossil Site"@en .
+
+gsqst:outcrop a skos:Concept ;
+    skos:altLabel "Dig site"@en ;
+    skos:broader gsqst:field-site ;
+    skos:definition "The part of a rock formation that appears at the surface of the ground. Source: Merriam-Webster Dictionary"@en ;
+    skos:inScheme gsqst:conceptScheme ;
+    skos:prefLabel "Outcrop"@en .
+
+gsqst:sediment-site a skos:Concept ;
+    skos:broader gsqst:field-site ;
+    skos:definition "A field site from which a sediment sample is taken or observation is made."@en ;
+    skos:inScheme gsqst:conceptScheme ;
+    skos:prefLabel "Sediment Site"@en .
+
+gsqst:soil-site a skos:Concept ;
+    skos:broader gsqst:field-site ;
+    skos:definition "A field site from which a soil sample is taken or observation is made."@en ;
+    skos:inScheme gsqst:conceptScheme ;
+    skos:prefLabel "Soil Site"@en .
+
+gsqst:stream-site a skos:Concept ;
+    skos:altLabel "Brook"@en,
+        "Canal"@en,
+        "Creek"@en,
+        "River"@en,
+        "Watercourse"@en ;
+    skos:broader gsqst:field-site ;
+    skos:definition "A field site from which a water sample is taken or observation is made."@en ;
+    skos:inScheme gsqst:conceptScheme ;
+    skos:prefLabel "Stream Site"@en .
+
+gsqst:base-station a skos:Concept ;
+    skos:definition "A  reference point used in a survey to measure a property, or provide a known reference quantity for survey calibration"@en ;
+    skos:inScheme gsqst:conceptScheme ;
+    skos:prefLabel "Base Station"@en ;
+    skos:topConceptOf gsqst:conceptScheme .
+
+gsqst:borehole a skos:Concept ;
+    skos:altLabel "Bore"@en,
+        "Core Hole"@en,
+        "Corehole"@en,
+        "Drillhole"@en,
+        "Well"@en ;
+    skos:definition "A Borehole is a permitted or actual drilled hole in the ground designed to exchange (or facilitate the exchange of) fluids between a subsurface reservoir and the surface (or another reservoir), or to enable the detection and measurement of rock properties.  A Borehole has a single Origin and all Wellbore(s) that extend from this Wll Origin are parts of the Borehole. Source: PPDM What Is A Well."@en ;
+    skos:inScheme gsqst:conceptScheme ;
+    skos:prefLabel "Borehole"@en ;
+    skos:topConceptOf gsqst:conceptScheme .
+
+gsqst:mine a skos:Concept ;
+    skos:altLabel "Pit"@en,
+        "Quarry"@en ;
+    skos:definition "A pit or excavation in the earth from which mineral substances are taken. Source: Merriam-Webster Dictionary"@en ;
+    skos:inScheme gsqst:conceptScheme ;
+    skos:prefLabel "Mine"@en ;
+    skos:topConceptOf gsqst:conceptScheme .
+
+gsqst:project-location a skos:Concept ;
+    skos:altLabel "Project"@en ;
+    skos:definition "The location of s scientific or technical survey, such as a geophysical survey area."@en ;

```

Comment:
The location of "A" scientific or

Reply:
fixed

Reply:
Should we just have a survey site that is the site of a survey activity rather than list all survey activities here?  Seems like duplication.